src/OVAL/results/oval_resultTest.c

```diff
@@ -564,6 +564,12 @@ static oval_result_t eval_item(struct oval_syschar_model *syschar_model, struct
 						oval_sysent_get_value(item_entity),
 						oval_sysitem_get_id(cur_sysitem), oval_state_get_id(state));
 			}
+			if (ent_val_res == OVAL_RESULT_ERROR) {
+				dI("Comparing entity '%s'='%s' of item '%s' to corresponding entity in state '%s' was not successful.",
```

Comment:
Hmm, actually you are right here I think. Because this is not an error in the tool but rather in the content. It's a gray area, `dI` is fine if you really meant it that way.